requirements.txt

```diff
@@ -1,2 +1,2 @@
 urllib3>=1.26.7
-PyU4V>=9.1.2.0
\ No newline at end of file
+PyU4V>=9.1.2.0
```

Comment:
As we have dropped the 91 support, then, can we make it 9.2.x.x?

Reply:
still supporting 9.1